Sources/Testing/Expectations/ExpectationChecking+Macro.swift

```diff
@@ -991,6 +992,7 @@ public func __checkClosureCall<R>(
     }
     mismatchExplanationValue = explanation
   } catch {
+    expression = expression.capturingRuntimeValues(error)
```

Comment:
Can you explain the effect of this? I wonder what will happen if the expression is comprised of multiple sub-expressions, won't the error be attributed to the first one even if it was a subsequent sub-expr which threw the error?

I'm mainly trying to get a sense of what the presentation of these errors will be, so an example or two would be great

Reply:
Ok I see, thanks